drivers/hmis/lib/form_data/tarrant_county/assessments/caseworker_exit.json

```diff
@@ -0,0 +1,611 @@
+          }
+        },
+        {
+          "text": "Phone Number",
+          "size": "SMALL",
+          "type": "STRING",
+          "link_id": "caseworker_exit_phone",
+          "component": "PHONE",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_phone"
+          }
+        },
+        {
+          "text": "Email",
+          "type": "STRING",
+          "link_id": "caseworker_exit_email",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_email"
+          }
+        },
+        {
+          "text": "Number of weeks in program",
+          "type": "INTEGER",
+          "required": true,
+          "link_id": "caseworker_exit_weeks_in_program",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_weeks_in_program"
+          }
+        },
+        {
+          "text": "Beginning GAS Score",
+          "helper_text": "Target 40.00",
+          "type": "NUMBER",
+          "link_id": "caseworker_exit_beginning_gas_score",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_beginning_gas_score"
+          }
+        },
+        {
+          "text": "Beginning GAS Score Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_beginning_gas_score_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_beginning_gas_score_date"
+          }
+        },
+        {
+          "text": "Exit GAS Score",
+          "helper_text": "Target 40.00",
+          "type": "INTEGER",
+          "link_id": "caseworker_exit_end_gas_score",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_end_gas_score"
+          }
+        },
+        {
+          "text": "Exit GAS Score Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_end_gas_score_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_end_gas_score_date"
+          }
+        },
+        {
+          "text": "Difference/Change in GAS score",
+          "type": "INTEGER",
+          "link_id": "caseworker_exit_gas_score_change",
+          "read_only": true,
+          "autofill_values": [
+            {
+              "formula": "caseworker_exit_end_gas_score - caseworker_exit_beginning_gas_score",
+              "autofill_when": [
+                {
+                  "operator": "EXISTS",
+                  "question": "caseworker_exit_end_gas_score",
+                  "answer_boolean": true
+                },
+                {
+                  "operator": "EXISTS",
+                  "question": "caseworker_exit_beginning_gas_score",
+                  "answer_boolean": true
+                }
+              ],
+              "autofill_behavior": "ALL",
+              "autofill_readonly": true
+            }
+          ],
+          "mapping": {
+            "custom_field_key": "caseworker_exit_gas_score_change"
+          }
+        },
+        {
+          "text": "Percent of GAS Change",
```

Comment:
Would it be desirable/worthwhile to create a new form component for percentages, so we can display a % symbol after this field?

Reply:
Yep sounds like a good idea. Can you add an issue to github?

Reply:
This would also work:
```
          "text": "<b>Change in GAS Score</b>: ${value}%",
          "type": "DISPLAY",
          "link_id": "caseworker_exit_gas_score_percent_change",
```
<img width="275" alt="Screenshot 2024-04-19 at 9 42 40 AM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/3b29993e-1072-4433-baf2-a243d42ec996">


Maybe we should use that approach for both difference and percentage, so they look consistent.

Reply:
This would work visually, but will the value still be saved as a custom data element? (as the ticket mentions "All fields should be saved. Including calculated fields like "difference between savings"") I can test this out to check.

Reply:
Here's what I found:
- If the field is display type, then the script for creating custom data element definitions will skip it, logging, "Skipping unexpected custom_field_key on non-question item. Unable to determine type."
- However, if the CDED already exists (we would have to create it manually), then the field value will get saved correctly when the form is submitted.

Reply:
Right. We have a few places where we actually store values on a DISPLAY type. I see how it's useful, but I don't love it, for this reason. It also complicates the form builder because the data type would need to be specified separately.

I think I'd prefer this type of thing (formatted value display) is implemented with something like `type: INTEGER, component: FORMATTED_DISPLAY, display_formula: "something: ${value}"`, or similar. I'll add it as a discussion topic to raise at standup.

I think for now lets leave what you have as-is.

Reply:
I think we never created a follow-up ticket for this idea -- typed display elements is a different issue from the integer/float issue captured here https://github.com/open-path/Green-River/issues/5962, if I am correctly untangling, right?